ballerina/tests/28_unary_basic_auth_client.bal

```diff
@@ -93,3 +93,22 @@ public isolated function testStringValueReturnWithBasicAuthWithEmpty() returns E
         test:assertFail(msg = "Expected grpc:Error not found.");
     }
 }
+
+@test:Config {enable: true}
+public isolated function testStringValueReturnWithBasicAuthWithInvalidHeader() returns Error? {
```

Comment:
Let's make test functions private.

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/333/commits/20aebae086f663da1d3f7264a7378682bc9af5c5